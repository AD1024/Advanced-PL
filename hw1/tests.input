1;; 2 +

3


;;

13;;

true && true ;;

false && true ;;

1 + 2 + 3 + 4 ;;

false && false && true && false;;

true + true + true;;

true && 1 + 2;;

1 && true + false;;

true && false && 1;;

x = 1 + 2 + 3;;
y = 114514;;
z = x + y;;
my_flag = true;;
false && my_flag;;
x + x + 1;;
x = x + y + z;;
my_flag = my_flag && true && false;;
my_flag;;
