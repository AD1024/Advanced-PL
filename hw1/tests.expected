1
5
13
bye!
